core/metadata.js

```diff
@@ -256,6 +257,12 @@ function isUpToDate (side /*: SideName */, doc /*: Metadata */) {
   return currentRev === lastRev
 }
 
+function isAtLeastUpToDate (side /*: SideName */, doc /*: Metadata */) {
+  let currentRev = doc.sides[side] || 0
+  let lastRev = extractRevNumber(doc)
+  return currentRev >= lastRev
```

Comment:
When I change `isUpToDate()` to `return currentRev >= lastRev` on master, tests (unit/integration/scenarios) pass on my laptop. I pushed #1276 to see what happens on CI.

Actually, all the conflict integration tests except one pass on master without any change.

The only test that fails (`local applied first -> conflict`) is effectively fixed by the new dissociation code in Merge, but it doesn't need the `isUpToDate()` change either.

Reply:
:thinking: Either I got confused, or it has more interactions with the #1274 than i thought. Let's merge that one first and then we will recheck.